src/backend/catalog/pg_type.c

```diff
@@ -493,7 +493,14 @@ TypeCreate(Oid newTypeOid,
  *
  * If rebuild is true, we remove existing dependencies and rebuild them
  * from scratch.  This is needed for ALTER TYPE, and also when replacing
+<<<<<<< HEAD
```

Comment:
Seems unresolved conflict

Reply:
this is part of upstream tag 
https://github.com/greenplum-db/gpdb/blob/5.29.8/src/backend/catalog/pg_type.c

Reply:
Let's ignore it, as a part of upstream work...